Taskfile.yaml

```diff
@@ -26,6 +26,7 @@ vars:
   TRIVY_VERSION: 0.55.0
   target: ""
   VALIDATION_FILES: "tools/validation/{main,messages,diff,no_cyrillic,doc_changes,copyright}.go"
+  SOURCE_REPO: '{{ .SOURCE_REPO | default "https://github.com" }}'
```

Comment:
Can we get situation when we should have to use tools from different sources?

Reply:
There is such situation with wget:
https://github.com/deckhouse/virtualization/pull/433/files#diff-8a7c2a335ed9a0f8ae5a03f8d901ac7db1c132af77b70f4f8b4151d68560c474R225